contracts/DebtLocker.sol

```diff
@@ -69,27 +78,27 @@ contract DebtLocker is IDebtLocker, DebtLockerStorage, MapleProxied {
         return _repossessed ? _handleClaimOfRepossessed() : _handleClaim();
     }
 
-    function setAllowedSlippage(uint256 allowedSlippage_) external override {
+    function setAllowedSlippage(uint256 allowedSlippage_) external override live {
```

Comment:
Setters should be allowed during paused state, since:
- there is to harm setting values during a paused state (and so no need to also waste gas during live states)
- the fix might require a setter, during a paused state